modules/apps/foundation/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp

```diff
@@ -552,7 +552,7 @@ name = HtmlUtil.escapeJS(name);
 							},
 							250
 						)
-					);
+					));
```

Comment:
SF -> following the above comment, we will need another line here as well, and an indentation for the code between 😉